src/containers/BookingForm.jsx

```diff
@@ -55,7 +56,8 @@ renderField.propTypes = {
   meta: PropTypes.object,
 }
 
-const renderSuccessMessage = bookingId => <h1>Booking Created with booking ID {bookingId}!</h1>
+const renderSuccessMessage = bookingId => <h1 name="result">Booking Created with booking ID {bookingId}!</h1>
+const renderErrorMessage = msg => <h1 name="error">Booking Failed: {msg}!</h1>
```

Comment:
i dunno whether adding name here is good.. no idea if error should be colored differently... also, don't know how to remove these when they click "book a room' again